contributions/api/controllers/contributions.py

```diff
@@ -844,6 +841,18 @@ def talents_get(token_info=None, id=None, talent_id=None):
 def ok_search():
     return rs_handlers.return_200("okay")
 
+def version_search():
+    contribution_yaml = open('contribution.yaml')
+    parsed_contribution_yaml = yaml.load(contribution_yaml, Loader=yaml.FullLoader)
+    return parsed_contribution_yaml['info']['version']
+
+def buildingblocks_search():
+    print(os.getcwd())
```

Comment:
Please remove the print statement or use the logger debug statement. Thanks.

Reply:
removed